syntax = "proto2";

package object_detection.protos;

import "object_detection/protos/model.proto";
import "object_detection/protos/train.proto";
import "object_detection/protos/eval.proto";
import "object_detection/protos/input_reader.proto";

// Top level configuration for FasterRCNNResnet101AstroNet.
message FasterRCNNResnet101AstroNet {
  optional DetectionModel model = 1;
  optional TrainConfig  train_config = 2;
  optional EvalConfig  eval_config = 3;
  optional InputReader train_input_reader = 4;
  optional InputReader eval_input_reader = 5;
}
